src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -35,12 +43,161 @@
 import org.bukkit.entity.Entity;
 import org.bukkit.inventory.ItemStack;
 
+import java.util.Collection;
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 
 public class MythicMobsBridge extends Bridge {
 
+    static class MythicMobsBridgeTags extends PseudoObjectTagBase<MythicMobsBridgeTags> {
+
+        public static MythicMobsBridgeTags instance;
+
+        public static void register() {
+            instance = new MythicMobsBridgeTags();
+            TagManager.registerStaticTagBaseHandler(MythicMobsBridgeTags.class, "mythicmobs", (t) -> instance);
+        }
+
+        @Override
+        public void registerTags() {
+
+            // <--[tag]
+            // @attribute <mythicmobs.item_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicItem IDs. See also <@link tag mythic_item>.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "item_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getItemNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.skills>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicSkill IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "skills", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getSkillNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicMob IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "mob_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getMobNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.active_mobs>
+            // @returns ListTag(MythicMobsMobTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all active MythicMobs on the server.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "active_mobs", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (ActiveMob entity : MythicMobsBridge.getMobManager().getActiveMobs()) {
```

Comment:
Pretty sure you don't need the class reference here? (as in, can just do `getMobManager().getActiveMobs()`)